.phpcs.xml.dist

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<ruleset
+		name="VIP Go Sites"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+	<description>Custom ruleset for VIP Go sites.</description>
+
+	<!-- Note: this file is only used for local PHPCS checks; the vip-go-ci bot does not read it. -->
```

Comment:
Let's include a reference to https://docs.wpvip.com/how-tos/customize-the-bot/ so that folks can more easily find how they can customise the bot's behaviour.